nucliadb_utils/nucliadb_utils/const.py

```diff
@@ -76,3 +76,4 @@ class Features:
     ROLLOVER_SHARDS = "nuclaidb_rollover_shards"
```

Comment:
oops! I just noticed this typo... we should be careful with this flag!

Reply:
actually, I can clean this up now. It was only temporary to get out to prod